Source/PINDiskCache.m

```diff
@@ -485,6 +491,11 @@ - (void)initializeDiskProperties
                 _metadata[key].size = fileSize;
                 byteCount += [fileSize unsignedIntegerValue];
             }
+
+            NSTimeInterval ageLimit;
```

Comment:
Added code to handle the failure.

So, I've spent some time doing some analysis and trying out different performance tweaks (which is why I haven't responded to your comments yet). Since PINDiskCache.ttlCache is not known at disk initialization, we cannot skip over the code which reads the extended file system attributes from disk.

There is a performance hit that will be incurred on cache initialization and is a function of the number of files in the cache. I tested adding 1000 objects into the disk cache and then compared initialization times between the current code and the code within this pull request. On average it took ~24ms to initialize the cache with the current code. On average it took ~35ms to initialize after the code changes, which is ~11ms/44% slower.

The slowdown might not be worth it for users that do not take advantage of the object-level age limits, so I have a proposal for mitigation:
- Make PINDiskCache.ttlCache (and PINDiskCache.ageLimit?) a readonly property (which will obviously break people who are currently using it).
- Add a ttlCache parameter to PINDiskCache's designated initializer.
- Skip over reading the extended file system attributes if ttlCache == NO. 

This change will basically have zero effect on users who are not using ttlCache.